src/fields/DateRangePicker/index.tsx

```diff
@@ -178,19 +179,53 @@ export function DateRangePicker({
       return
     }
 
-    const startUtcDate = selectedStartDateAsDayjsRef.current.toDate()
-    const endUtcDate = selectedEndDateAsDayjsRef.current.toDate()
+    const startUtcDate = selectedStartDateTimeAsDayjsRef.current.toDate()
+    const endUtcDate = selectedEndDateTimeAsDayjsRef.current.toDate()
 
     const nextDateRange: DateRange = [startUtcDate, endUtcDate]
 
     ;(onChange as (nextUtcDateRange: DateRange) => Promisable<void>)(nextDateRange)
   }, [isStringDate, onChange])
 
-  const closeRangeCalendarPicker = useCallback(() => {
-    isRangeCalendarPickerOpenRef.current = false
+  const callOnChangeUndefinedIfInputsAreEmpty = useCallback(() => {
```

Comment:
This function content has not been changed, it was just renamed from `handleDateOrTimeInputInput()` to `callOnChangeUndefinedIfInputsAreEmpty()`.